sql/feedback.sql

```diff
@@ -0,0 +1,31 @@
+DROP TABLE IF EXISTS complaint_emails;
+DROP TABLE IF EXISTS complaint_messages;
+DROP TABLE IF EXISTS complaint_info;
+
+CREATE TABLE complaint_info (
+  complaint_id INT(11) NOT NULL AUTO_INCREMENT,
+  subject VARCHAR(50) DEFAULT NULL,
+  status VARCHAR(36) DEFAULT NULL,
+  uuid BINARY(16) NOT NULL,
+  org VARCHAR(10) NOT NULL,
```

Comment:
Would be more efficient to store an `INT`; could define an `IntEnum` in Python that defines the different `org` values